lib/membrane/rtp/silence_discarder.ex

```diff
@@ -35,15 +30,15 @@ defmodule Membrane.RTP.SilenceDiscarder do
               ],
               silence_threshold: [
                 spec: 1..127,
-                default: 127,
+                default: 64,
```

Comment:
Why did you change that to 64? It can drop the normal speech then, 127 means silent.

Reply:
I explained in PR description, 127 would not drop anything in normal cases as there is always some ambient noise. Do we want to target only explicitly muted streams? If so, we can change it back. If we want to also consider some amount of ambient noise, we can think about a more safe default value.

Reply:
I think that for now we are good with dropping just the ambient noise (unless you want to spend more time experimenting with what audio level can be considered silence). The main case is when there are a lot of people in the room and most of them are muted but the packets still get sent, it is pretty much the main reason for this filter to exist.